src/pages/team/index.tsx

```diff
@@ -131,44 +136,52 @@ const Team = ({team: teamData}: TeamPageProps) => {
     }
   }, [teamDataNotAvailable])
 
+  if (teamData === null) return null
+
   return (
     <LoginRequired>
-      {!!teamData && (
```

Comment:
Moving the presence check up here to reduce nesting and help with some TS type narrowing.